src/core/prompts/sections/markdown-formatting.ts

```diff
@@ -0,0 +1,32 @@
+export function markdownFormattingSection(): string {
+	return `====
+
+MARKDOWN FORMATTING
+
+When providing responses, use proper markdown formatting:
+
+- Use code blocks with language hints for code snippets:
+  \`\`\`typescript
+  const example = "code here";
+  \`\`\`
+
+- Use bullet points for lists
+- Use headings with appropriate levels (# H1, ## H2, etc.)
+- Format inline code with backticks and clickable links with line numbers
+- Use tables when comparing or listing structured data
+- Include line breaks between sections for readability
+
+When referencing code, you MUST ALWAYS use clickable links with backticked definition names and line numbers:
+
+- When referencing functions or other major code definitions in explanatory text, ALWAYS use the format [\`entityName()\`](filepath:line) to create clickable links
+- Every function, structure, variable, property, field, or other major code entity that you mention MUST have a corresponding [\`link\`](to/the/file.ext:line) during any human-readable markdown response
```

Comment:
It would be nice to visually distinguish code blocks that are links from code blocks that are not, cc: @sachasayan

Reply:
@mrubens  Elaborate? Do you mean hard assets (files) vs ephemeral fragments?

Reply:
I just mean that we render `code` and [`code`](https://www.somewhere.com) identically in our markdown renderer

Reply:
Can't tell at a glance if a code formatted string is clickable

Reply:
good feedback I will take a look

Reply:
fixed in 7c83a56b using vscode-theme-defined colors:

![image](https://github.com/user-attachments/assets/ff521bd0-3060-4325-8620-81b362f0ea22)